modules/server/src/main/ts/bedrock/compiler/Imports.ts

```diff
@@ -15,17 +15,28 @@ const filePathToImport = (useRequire: boolean, scratchFile: string) => {
     filePath = filePath.replace(/\\/g, '/');
 
     return [
-      useRequire ? `require("${relativePath}");` : `import "${relativePath}";`, // rollup doesn't support require
-
-      `addTest("${filePath}");`
+      'try {',
```

Comment:
How interesting. Is "```" an option?

Reply:
More than likely, this was probably from before ES 6 was possible on node and has just been adjusted slowly over time.